tests/api/resource_test.py

```diff
@@ -268,14 +268,6 @@ def test_get_run_from_identifier_number(self):
         assert_equal(job_run, self.job_runs.get_run_by_num.return_value)
         self.job_runs.get_run_by_num.assert_called_with(3)
 
-    def test_get_run_from_identifier_state_name(self):
-        job_run = self.resource.get_run_from_identifier('SUCC')
```

Comment:
Huh. I kinda wonder what the original authors' intent was with this. 
I agree that I think we can safely get rid of it.